schedule/yast/skip_registration/skip_registration_15sp3.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name:           skip_registration
+description:    >
+  Skipping registration for SLE 15, as requires network connection.
+  This is default behavior for SLE 12.
+vars:
+  SCC_REGISTER: 'none'
+  ADDONS: 'desktop'
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/skip_registration
+  - installation/addon_products_sle
```

Comment:
The whole point of creating a new schedule is that there is no libuy module that works for this page on 15SP3 and I am not sure how to make one since the typical POST actions don't work.

Reply:
Joaquin, we were together yesterday at the office when I showed you that it doesn't work and you said that indeed libyui-rest-api 15SP3 had some issue that the fix might not be backported. Am I having false memories?

Reply:
I was referring to SP3, that we were producing the error ourselves sending the wrong rest command and perhaps it is not able to handle, when you send the same wrong commands to SP5 might be able...or maybe not, just that the channels are wrong and that error will hit first.
But when we will send the right REST command then it should be ok, which is what you are trying to do here.
What I saw is that you were trying to use a test module that only works for a registered system.

Reply:
mmm but actually the modules is ok, `select_module_desktop` should try to find addons_repos and doesn't find it.... let me check that more in detail.

Reply:
the only explanation is that doesn't depend on the registration itself that appears one screen or the other, but the medium, as it is Full we might need `register_module_desktop` instead.

Reply:
I have tried to send http://localhost:39113/v1/widgets?action=select&id=items&value=sle-module-desktop-applications-15.6-x86_64 as the register_module_desktop does and it was not found as expected

Reply:
you are right, so you are using the right module the the curl command doesn't work, just reproduce it, then could you please check with Ladislav about that? looks like the fix wasn't backported (another one that the one I was referring yesterday, sorry for the confusion).

Reply:
it should be `id=addon_repos&action=check&value=Desktop+Applications+Module'` and doesn't work as you said, strange, I guess at that point we didn't have all the test suite in libyui-rest-api. Check the version of the libyui-rest-api in order to communicate with developer. Depends if it is easy to backport we might have it, but it might happen that doesn't worth and then your solution....is ok, I mean still we could consider to do some hybrid stuff, but idk.

Reply:
Ok, I will try to check the merged PRs and let him know

Reply:
Now that I remembered, in 15SP3 there was this annoying feature, to click on the module name and see the description without checking the box, maybe that's what 's causing the issue